oaknational/Oak-Web-Application
# fix(analytics): only fire single page view

## Description

- previously we were firing double page views because useEffect callback was being called again (provider was remounting) 


## How to test

1. Go to {cloud link}
2. browse about with dev console open
3. You should see single pageview events fire per page

## Checklist
